ext/mimemagic/Rakefile

```diff
@@ -3,7 +3,15 @@ require "rake/clean"
 
 def locate_mime_database
   possible_paths = [
-    (File.expand_path(ENV["FREEDESKTOP_MIME_TYPES_PATH"]) if ENV["FREEDESKTOP_MIME_TYPES_PATH"]),
+    (File.expand_path(ENV["FREEDESKTOP_MIME_TYPES_PATH"]) if ENV["FREEDESKTOP_MIME_TYPES_PATH"])
+  ] + (
+    if ENV["XDG_DATA_DIRS"]
```

Comment:
[suggestion]
If you use ENV.fetch with [] as default you can avoid uma extra if statement. Check it [here](https://ruby-doc.org/core-2.5.0/ENV.html#method-c-fetch).